package.json

```diff
@@ -21,7 +21,11 @@
     "kibble:build": "kibble publish",
     "build": "npm-run-all css js:build css:build kibble:build",
     "language:csv": "node scripts/language-file-to-csv/language-file-to-csv.js site/en_AU.all.json",
-    "language:validate": "node scripts/language-file-validator.js"
+    "language:validate": "node scripts/language-file-validator.js",
+    "version": "node scripts/core-template-version.js",
+    "prestart": "npm run version",
+    "prebuild": "npm run version",
```

Comment:
I can't find this in [the lifecycle scripts](https://docs.npmjs.com/cli/v7/using-npm/scripts) - is `prebuild` legit?

Reply:
It's a [user-defined life cycle script](https://docs.npmjs.com/cli/v7/using-npm/scripts#npm-run-user-defined).